doc/modules/linear_model.rst

```diff
@@ -61,6 +61,12 @@ example, when data are collected without an experimental design.
 
    * :ref:`sphx_glr_auto_examples_linear_model_plot_ols.py`
 
+Ordinary Least Squares uses a Singular Value Decomposition (SVD) based
+approach. The ``LinearRegression`` class has an additional, optional
+``solver`` parameter, which if set to ``"cholesky"`` uses the Cholesky
+factorization instead. See `these notes
+<https://www.cs.ubc.ca/~schmidtm/Courses/540-F14/leastSquares.pdf>` for a
```

Comment:
It might be better to place references under a `.. topic:: References` section. One very good reference for least squares, in my opinion, is https://www.math.uchicago.edu/~may/REU2012/REUPapers/Lee.pdf.

Reply:
```suggestion
<https://www.cs.ubc.ca/~schmidtm/Courses/540-F14/leastSquares.pdf>`__ for a
```
correct web link syntax at a minimum